controllers/lectureController.js

```diff
@@ -86,4 +89,28 @@ exports.getLecturesById = async(req, res, next) => {
 
     })
 
+}
+
+// Api to delete lecture using lecture id
+exports.deleteLectureById = async(req, res, next) => {
+    var getLetcureById = await Lecture.findById(req.param('id'), (err, result) => {
+        if (err) {
+            res.send(err);
+        }
+        if (result) {
+            try {
+                Lecture.deleteOne({
+                    _id: new mongo.ObjectID(req.param('id'))
+                }, (error) => {
+                    if (error) throw error;
+                });
+                res.send(JSON.stringify("Record deleted successfully!"));
```

Comment:
Instead of using `res.send(` and sending a stringified version of JSON. Please use `res.json()`

Reply:
Done